samples/ingestion/ingestion-client/FetchTranscription/TranscriptionProcessor.cs

```diff
@@ -323,21 +329,21 @@ private async Task WriteErrorReportAsync(string errorString, string jobName)
 
             await this.storageConnector.WriteTextFileToBlobAsync(
                 errorString,
-                FetchTranscriptionEnvironmentVariables.ErrorReportOutputContainer,
+                this.appConfig.ErrorReportOutputContainer,
                 errorTxtname).ConfigureAwait(false);
         }
 
         private async Task ProcessSucceededTranscriptionAsync(string transcriptionLocation, string subscriptionKey, TranscriptionStartedMessage serviceBusMessage, string jobName, ILogger log)
         {
             log.LogInformation($"Got succeeded transcription for job {jobName}");
 
-            var transcriptionAnalyticsOrchestrator = new TranscriptionAnalyticsOrchestrator(serviceBusMessage.Locale, log);
+            var transcriptionAnalyticsOrchestrator = new TranscriptionAnalyticsOrchestrator(serviceBusMessage.Locale, log, Options.Create(this.appConfig));
```

Comment:
why do you use Options.Create(this.appConfig) here instead of just passing the appConfig?

Reply:
`Options.Create(this.appConfig)` wraps the `appConfig` to `IOptions<AppConfig>` which is expected as a constructor parameter for `TranscriptionAnalyticsOrchestrator`.
Directly passing appConfig breaks the pattern of IOptions which is now being used for managing the configuration.
Also eventually we should also move `TranscriptionAnalyticsOrchestrator` to be used as a dependency. When that happens we won't need to pass the appConfig.